examples/gitserver/README.md

```diff
@@ -38,6 +38,8 @@ Prerequisites:
     $ oc policy add-role-to-user edit -z git
     ```
 
+**NOTE:** To deploy the git server on OpenShift Online, you must switch to using persistent
+storage. See the *Persistent Storage* section below.
```

Comment:
Per feedback from @csrwng it looks like git truncated part of my last response via embedded email.

Reposting:
1) Per direction from Dan we went with a 1 G PV default across all templates, so doing so here would be consistent.
2) I think our users are already use to the ephemeral/persistent naming convention, so I don't think that should be concern